rerun_py/rerun_sdk/rerun/notebook.py

```diff
@@ -144,20 +147,26 @@ def __init__(
         self._viewer = _Viewer(
             width=width if width is not None else _default_width,
             height=height if height is not None else _default_height,
+            url=url,
         )
 
         if use_global_recording:
             recording = get_data_recording(recording)
             if recording is None:
-                raise ValueError("No recording specified and no active recording found")
-
-            bindings.set_callback_sink(
-                recording=recording.to_native(),
-                callback=self._flush_hook,
-            )
-
-            if blueprint is not None:
-                recording.send_blueprint(blueprint)
+                if url is None:
```

Comment:
This all feels very wrong, but not because of what you added. The whole `use_global_recording` thing is not working as advertised at all. I can make a separate PR to fix this afterwards

Reply:
ohhhhh seeing it as well now. Yeah that parameter is super broken Oo. Thanks